src/components/MoneyReportHeader.js

```diff
@@ -76,18 +76,19 @@ function MoneyReportHeader({session, personalDetails, policy, chatReport, nextSt
     const policyType = lodashGet(policy, 'type');
     const isPolicyAdmin = policyType !== CONST.POLICY.TYPE.PERSONAL && lodashGet(policy, 'role') === CONST.POLICY.ROLE.ADMIN;
     const isManager = ReportUtils.isMoneyRequestReport(moneyRequestReport) && lodashGet(session, 'accountID', null) === moneyRequestReport.managerID;
-    const isPayer = policyType === CONST.POLICY.TYPE.CORPORATE ? isPolicyAdmin && isApproved : isPolicyAdmin || (ReportUtils.isMoneyRequestReport(moneyRequestReport) && isManager);
```

Comment:
I removed the condition `isApproved` since now admins can Pay a request by skipping the `Approve` state in paid policies (control and collect).